### PR TITLE
add a command line flag to specify description file for package

### PR DESCRIPTION
**Pull request by keertip**
@sethladd

### CODE REVIEW

File: bin/dartdoc.dart [hunk 1]
```diff
@@ -68,5 +69,7 @@ ArgParser _createArgsParser() {
   parser.addFlag('sdk-docs', help: 'generate docs for the Dart SDK.'
       ' '
       'Use "--dart-sdk" option to specify path to sdk');
+  parser.addOption('readme',
+      help: 'path to the package description file, including file name');
```
Comment (sethladd) [inline bin/dartdoc.dart]: "package or SDK" ?

I think "including file name" is probably redundant.

File: lib/src/package_utils.dart [hunk 2]
```diff
@@ -20,7 +20,11 @@ Map _getPubspec(String directoryName) {
   return loadYaml(contents);
 }
 
-String getPackageDescription(String directoryName) {
+String getPackageDescription(bool isSdk, String readmeLoc, String directoryName) {
+  if (isSdk && readmeLoc !=  null) {
+    var readmeFile = new File(readmeLoc);
+    if (readmeFile.existsSync()) return readmeFile.readAsStringSync();
```
Comment (sethladd) [inline lib/src/package_utils.dart]: if it doesn't, what happens? when do we tell the user the filename that they provided is invalid?

Comment (keertip) [inline lib/src/package_utils.dart]: Missed that. Will add check.

File: bin/dartdoc.dart (hunk 1 above)
Comment (keertip) [inline bin/dartdoc.dart]: changed to SDK

File: test/fake_package/test_readme.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Welcome to the Dart API reference documentation.
```
Comment (sethladd) [inline test/fake_package/test_readme.md]: Maybe include a bit of markdown syntax here so we can verify it's getting converted correctly?

**Comment by sethladd** [comment]
lgtm with one small suggestion

File: test/fake_package/test_readme.md (hunk 3 above)
Comment (keertip) [inline test/fake_package/test_readme.md]: done

**Pull request merged**